config/routes.rb

```diff
@@ -28,6 +28,7 @@
   get "/find-local-council/:authority_slug" => "find_local_council#result"
 
   get "/foreign-travel-advice", to: "travel_advice#index", as: :travel_advice
+  get "/foreign-travel-advice/:country/*slug", to: "travel_advice#show"
```

Comment:
Hmm, could we call the part something other than slug? Slug's already got too many meanings, and I think in this case we're going to end up overloading it.

Reply:
Looking through the code I think it's best to keep this as "slug". It's called "slug" in the content item, and I think the part and part helper code will become very confusing if we're using two different terms to refer to the same thing. I think it makes more sense in the context of travel-advice pages for the "slug" to refer to the sub-page of a country, and for it to mean that everywhere.